### PR TITLE
JavaScript: Prefer a trailing comment when there is comment between test and consequent on ternary

### LINKED ISSUE #2076: Comment inside multiline ternary moves after `?`
See https://prettier.github.io/prettier/#%7B%22content%22%3A%22const%20resolver%20%3D%20options.resolver%5Cn%20%20%2F*%20%24FlowFixMe%20*%2F%5Cn%20%20%3F%20require(options.resolver)%5Cn%20%20%3A%20defaultResolver%3B%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%7D%7D

Ref  facebook/jest#3780

### PR DESCRIPTION
**Pull request by sosukesuzuki**
Fix #2076 

- [x] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [x] (If the change is user-facing) I’ve added my changes to the `CHANGELOG.unreleased.md` file following the template.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by j-f1** [comment]
What’s the status of this PR?

**Comment by fisker** [comment]
@sosukesuzuki

File: src/main/comments.js [hunk 1]
```diff
@@ -214,6 +214,17 @@ function attach(comments, ast, text, options) {
 
     const isLastComment = comments.length - 1 === i;
 
+    const isEnclosedByTernary =
```
Comment (thorn0) [inline src/main/comments.js:217]: This is not the right file for JS-specific things. This should be handled in `language-js/comments.js`.

**Comment by sosukesuzuki** [comment]
ooops sorry I forgot. I'll work on this at today or tomorrow.

**Comment by sosukesuzuki** [comment]
It is hard to merge `master`  into this branch because `master` includes may changes. I'll close this PR, for now and I'll open new PR to fix #2076

**Pull request closed without merging**